.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
curious why this is needed

Reply:
The new Travis default build environment 'xenial' is being rolled out (https://changelog.travis-ci.com/xenial-as-the-default-build-environment-99476), which does not seem to support oraclejdk8 (at least not in the 'standard' way). 
Some more details are in this thread: https://travis-ci.community/t/the-travis-ci-build-could-not-be-completed-due-to-an-error/1345